lib/metasploit/framework/data_service/remote/http/remote_async_callback_service.rb

```diff
@@ -0,0 +1,17 @@
+require 'metasploit/framework/data_service/remote/http/response_data_helper'
+
+module RemoteAsyncCallbackDataService
+  include ResponseDataHelper
+
+  ASYNC_CALLBACK_API_PATH = '/api/v1/async-callbacks'
+  ASYNC_CALLBACK_MDM_CLASS = 'Mdm::AsyncCallback'
+
+  def async_callbacks(opts)
+    path = get_path_select(opts, ASYNC_CALLBACK_API_PATH)
+    json_to_mdm_object(self.get_data(path, nil, opts), ASYNC_CALLBACK_MDM_CLASS, [])
```

Comment:
The method signature to `json_to_mdm_object` recently changed in #11948 and no longer has a 3rd parameter.
```
def json_to_mdm_object(response_wrapper, mdm_class)
```

Reply:
Good catch, thanks!